receiver/otlpreceiver/otlp_test.go

```diff
@@ -240,74 +214,6 @@ func testHTTPJSONRequest(t *testing.T, url string, sink *internalconsumertest.Er
 		}
 		require.Len(t, allTraces, 0)
 	}
-
-}
-
-func TestJsonMarshaling(t *testing.T) {
-	m := jsonpb.Marshaler{}
-	json, err := m.MarshalToString(&resourceSpansOtlp)
-	assert.NoError(t, err)
-
-	var resourceSpansOtlp2 otlptrace.ResourceSpans
-	err = jsonpb.UnmarshalString(json, &resourceSpansOtlp2)
-	assert.NoError(t, err)
-
-	assert.EqualValues(t, resourceSpansOtlp, resourceSpansOtlp2)
-}
-
-func TestJsonUnmarshaling(t *testing.T) {
```

Comment:
These are important tests that verify JSON marshaling. Why delete these?

Reply:
We are verifying that jsonpb which is from proto does the right thing... We should not unittest third party libraries

Reply:
Friendly ping @tigrannajaryan